conan/cli/commands/install.py

```diff
@@ -33,6 +33,8 @@ def install(conan_api, parser, *args):
                         help='Deploy using the provided deployer to the output folder')
     parser.add_argument("--deployer-folder",
                         help="Deployer output folder, base build folder by default if not set")
+    parser.add_argument("--deploy-package", action="append",
```

Comment:
While I understand that `--deploy-package` gives a better meaning of performing an action, I think that having symmetric options with the two options above feels cleaner? My suggestion would thus be `--depolyer-package`

Reply:
Yes, I think that's a valid point, not having to remember if it was _deploy_ or _deployer_ everytime so for me it's fine `--deployer-package`

Reply:
Sure, lets change it

Reply:
Done